### ISSUE TITLE
Loading

### ISSUE THREAD

**Issue by annevk** [WICG/webpackage#115]
The README suggests loading will be done at the W3C, which made me wonder. Shouldn't loading be integrated into Fetch somehow?

**Comment by annevk** [comment]
(This concern also applies to some extent to response header processing, which also takes place in Fetch and has been a source of confusion for IETF efforts as they don't carefully consider service workers, synthetic responses, etc.)

**Comment by jyasskin** [comment]
Yes, and sorry for conflating the W3C and the WHATWG. I expect to do the initial web-side spec in the WICG, and then merge into Fetch, Service Workers, and whichever other web specs make sense, in those specs' organizations.
